packages/worker/src/api/routes/global/users.ts

```diff
@@ -108,6 +108,11 @@ router
     buildInviteMultipleValidation(),
     controller.inviteMultiple
   )
+  .post(
```

Comment:
This should be `DELETE /api/global/users/multi/invite`

Reply:
That's what I thought initially, but there seems to be a lot of debate whether you should do it or not: https://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request

There is one place in which we pass a body into a DELETE call, but it's actually a copy+paste mistake, as we only use the ID. 

```ts
const response = await api.del(
    `/api/system/accounts/${account.accountId}/metadata`,
    {
      body: account,
      headers: {
        [Header.API_KEY]: env.BUDIBASE_API_KEY,
      },
    }
  )
```

```ts
export const destroy = async (ctx: any) => {
  const accountId = accounts.metadata.formatAccountMetadataId(
    ctx.params.accountId
  )
  await accounts.metadata.destroyMetadata(accountId)
  ctx.status = 204
}
```

I'll fix the above at least, and happy to use delete on this endpoint, just not sure what the best practice is.

Reply:
From what I can find the body of a delete should not identify the resources being deleted: https://www.mscharhag.com/api-design/rest-deleting-resources

Reply:
It looks like an edge case on the standard, but plenty of API uses them. Personally, I think that keeping the "delete" verbs makes it safer and more verbose

Reply:
Sure! I'll make change 🙂

Reply:
Well, if we want to follow the REST standard:

DELETE:
- Idempotent: Yes
- Safe: No

POST:
- Idempotent: No
- Safe: No

So here we are creating an idempotent POST method because in the end it is covering a delete under the hood; no matter how many times you make the POST.

Furthermore, as you say, normally DELETE shouldn't contain that kind of information.

Anyway, here we are just discussing about the standard HTTP spec and (as far as I understand you can implement REST following a different RFC); REST principles are not restricted to an specific protocol.

Said that I think currently there isn't an standard solution. Well we can define our own RCF like the teapot protocol 🤣 -> https://datatracker.ietf.org/doc/html/rfc2324 IMO we can go ahead with POST.

Reply:
So... Nothing to discuss 💩